lms/product/moodle/_plugin/course_copy.py

```diff
@@ -0,0 +1,26 @@
+from lms.product.plugin.course_copy import CourseCopyFilesHelper, CourseCopyGroupsHelper
+from lms.services.moodle import MoodleAPIClient
+
+
+class MoodleCourseCopyPlugin:
+    file_type = "moodle_file"
+
+    def __init__(
+        self,
+        api: MoodleAPIClient,
+        groups_helper: CourseCopyGroupsHelper,
+    ):
+        self._api = api
+        self._groups_helper = groups_helper
+
+    def find_matching_group_set_in_course(self, course, group_set_id):
+        return self._groups_helper.find_matching_group_set_in_course(
```

Comment:
This works somehow magically because we already store group sets in the course when listing them